### DISCUSSION TITLE
用夜神模拟器的朋友们，你们玩碧蓝航线帧率表现如何

### DISCUSSION THREAD

**Discussion by Altria17388** [LmeSzinc/AzurLaneAutoScript#2138] [General]
看到模拟器简介里推荐夜神模拟器，专门下了一个试试，结果跑碧蓝航线帧率极不稳定，复杂场景甚至直接掉到三四十帧。然后又下了一个蓝叠国际版试试，结果蓝叠莫名其妙会偶尔导致某一段时间内鼠标卡得无法自理，直接弃用。电脑配置12600KF+3070。同样情况下用雷电模拟器开120帧，帧数很丝滑，只不过怕雷电模拟器容易出问题，目前不得不先用夜神模拟器

**Comment by xhqss** [comment]
用海外版的蓝叠？

**Reply by Altria17388** [reply]
国际版的蓝叠我用起来会遇到卡鼠标的问题，具体现象就是模拟器刚打开加载的时候，鼠标经常会变得巨卡，感觉太影响使用了

**Reply by Altria17388** [reply]
不过我现在换到逍遥模拟器了，用着暂时没啥问题，帧数基本能稳定50-60帧
